graphql/src/main/java/com/linecorp/armeria/common/graphql/scalar/MoreScalars.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.graphql.scalar;
+
+import com.linecorp.armeria.common.multipart.MultipartFile;
+
+import graphql.schema.GraphQLScalarType;
+
+/**
+ * This contains the implementations of the custom Scalar types.
+ */
+public final class MoreScalars {
```

Comment:
Let's add `@UnstableApi`.
How about also renaming to `ArmeriaScalarTypes`?

Reply:
Some content conflicts with [this comment](https://github.com/line/armeria/pull/4199#discussion_r851573990
).

Reply:
Then, let's just add `@UnstableApi`. 😆